features/DOS/ssp_digital_outcomes_service.feature

```diff
@@ -45,11 +45,11 @@ Feature: Submitting a new DOS service for Digital outcomes
 
     #Service can't be completed unless at least one "Team capability" has been defined
     And The 'Mark as complete' button should not be on the page
-    #And The service 'can't' be marked as complete
 
   Scenario: A draft service has been created
     Given I am at '/suppliers/frameworks/digital-outcomes-and-specialists/submissions'
-    Then There is 'a' draft 'Apply to provide teams for digital outcomes' service
+    Then There is 'a' draft 'digital outcomes' service
```

Comment:
Much better to get this back like this - sorry I hacked in the long version to get the tests working prior to DOS launch.